gazelle/python/file_parser.go

```diff
@@ -41,7 +41,7 @@ const (
 
 type ParserOutput struct {
 	FileName string
-	Modules  []module
+	Modules  []Module
 	Comments []comment
```

Comment:
nit: I think that all of the types in this public `ParserOutput` type should be public if the value can be accessed, hence I think the `comment` should be also exposed.

Could you please do that as well? This is a non-functional change and having a git history where such refactors are grouped together may make it easier to read. Having in the CHANGELOG a line that says something like:
> (gazelle) Types for exposed members of `python.ParserOutput` are now all public.